Specifications/Applications/for_BoundedContext/when_creating.cs

```diff
@@ -4,11 +4,11 @@ namespace Dolittle.Applications.Specs.for_BoundedContext
 {
     public class when_creating
     {
-        const string name = "Some Bounded Context";
+        static BoundedContextName name = new BoundedContextName{Value = "Some bounded context"};
         static BoundedContext bounded_context;
 
         Because of = () => bounded_context = new BoundedContext(name);
 
-        It should_have_the_name = () => ((string)bounded_context.Name).ShouldEqual(name);
+        It should_have_the_name = () => ((string)bounded_context.Name.AsString()).ShouldEqual(name.Value);
```

Comment:
Should just be able to equate the concepts.  So boundedContext.Name.ShouldEqual(name).  Implicits don't work with the fluent interface but you have the BoundedContextName in the variable.